src/core/TrackContainer.cpp

```diff
@@ -124,6 +122,9 @@ void TrackContainer::loadSettings( const QDomElement & _this )
 		if( node.isElement() &&
 			!node.toElement().attribute( "metadata" ).toInt() )
 		{
+			QString trackName = node.toElement().hasAttribute( "name" ) ? node.toElement().attribute( "name" ) :
+															   node.firstChild().toElement().attribute( "name" );
```

Comment:
125 and 126 too long lines. The [coding conventions](https://github.com/LMMS/lmms/wiki/Coding-conventions) suggestion for ternary operators is something like this:

```
QString trackName = node.toElement().hasAttribute( "name" )
            ? node.toElement().attribute( "name" )
            : node.firstChild().toElement().attribute( "name" );
```